app/controllers/api/saved_jobs_controller.rb

```diff
@@ -26,6 +26,15 @@ def show
     end
   end
 
+  def destroy
+    job_connection = SavedJob.joins(:job).find_by("jobs.job_key = '#{params[:key]}'", "user_id = #{current_user.id}")
```

Comment:
This seems really complex and includes sql injection issues.  (params direct into a sql statement are a major no no" Instead use ? mark as a placeholder and the arity of the find_by function will increase per ?